### PR TITLE
Port tests in test_transforms_video.py to pytest

### PR DESCRIPTION
**Pull request by DevPranjal**
Refactor test_transforms_video.py to pytest as mentioned in #4033

### CODE REVIEW

File: test/test_transforms_video.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def test_center_crop_video(self):
 
         msg = "height: " + str(height) + " width: " \
             + str(width) + " oheight: " + str(oheight) + " owidth: " + str(owidth)
-        self.assertEqual(sum1.item() > 1, True, msg)
+        assert sum1.item() > 1 == True, msg
```
Comment (DevPranjal) [inline test/test_transforms_video.py:85]: `flake` test here suggests:
> `./test/test_transforms_video.py:85:32: E712 comparison to True should be 'if cond is True:' or 'if cond:'`
Do I need to change or stick with the original implementation?

Comment (NicolasHug) [reply test/test_transforms_video.py:85]: Yes we should just use `assert sum1.item() > 1, msg` :)

**Comment by NicolasHug** [comment]
Thanks @DevPranjal !!

**Comment by DevPranjal** [comment]
Thanks for the fix.

**Pull request merged**